tests/rt.sh

```diff
@@ -235,8 +235,13 @@ elif [[ $MACHINE_ID = hera.* ]]; then
   PYTHONHOME=/scratch1/NCEPDEV/nems/emc.nemspara/soft/miniconda3_new_20210629
   export PATH=$PYTHONHOME/bin:$PATH
   export PYTHONPATH=$PYTHONHOME/lib/python3.7/site-packages
-  ECFLOW_START=$PYTHONHOME/bin/ecflow_start.sh
-  ECF_PORT=$(( $(id -u) + 1500 ))
+
+  if [[ ! $HOSTNAME = hecflow* ]]; then
+    echo "ERROR: On Hera we must be logged into 'hecflow' login nodes"
+    exit 1
+  fi
```

Comment:
This check should be done only if ecflow is used, otherwise rt.sh should be allowed to run on all nodes.

Reply:
Yes, for now it's a temporary workaround. It would be best to implement this in the autoRT only.

Reply:
Why is it temporary? Are we going to allow users to use ecflow on all nodes?

Reply:
I actually got error yesterday when I do "rt.sh -l rt.conf1", so it's good to add a condition here just for ecflow.

Reply:
It is temporary in the sense it will not force all Hera things to be on hecflow01. I will move the code into (or check using the) -e option check.